magefile.go

```diff
@@ -120,16 +176,38 @@ func CheckNoBeatsDependency() error {
 	return nil
 }
 
-// CheckModuleTidy checks if `go mod tidy` was run before.
-func CheckModuleTidy() error {
+// CheckModuleTidy checks if `go mod tidy` was run before the last commit.
+func (Deps) CheckModuleTidy() error {
 	err := sh.Run("go", "mod", "tidy")
 	if err != nil {
 		return err
 	}
-	err = sh.Run("git", "diff", "--exit-code")
+	err = sh.Run("git", "diff", "--exit-code", "go.mod")
```

Comment:
This was a bug, any change in the repo caused the failed check and this misleading error message.